api/controllers/apps_test.go

```diff
@@ -47,7 +49,10 @@ func TestAppShow(t *testing.T) {
 }
 
 func TestAppShowWithAppNotFound(t *testing.T) {
-	aws := test.StubAws(test.DescribeStackNotFound("bar"))
+	aws := test.StubAws(
+		test.DescribeStackNotFound("bar"),
```

Comment:
I think changing the order would also address up a few of the test cycle questions I have.